README.md

```diff
@@ -1492,22 +1586,13 @@ env_files = [
 ]
 ```
 
-In this example, **profile.env** is only loaded in case the runtime profile is **development**.<br>
-*More on profiles in the [profiles](#usage-profiles) section.*<br>
-<br>
-Relative paths are relative compared to the toml file that declared them and not to the current working directory.<br>
-<br>
-The same **env_files** attribute can be defined on the task level, however relative paths on the task level are relative to the current working directory.<br>
```

Comment:
why is this block removed?

Reply:
It wasn't. Just reworded into:

Paths to environment files can also be defined globally in the `env_files` key of the `Makefile.toml`, which will be loaded in the order they are defined. All relative paths are relative to the directory containing the `Makefile.toml` they were defined in.

> **Note:** `env_files` can also be used on a task level, be aware that relative paths will instead be relative **to the current working directory**